packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -49,29 +50,29 @@ export default class ComponentFile {
     };
 
     const filepath = this.studioSourceFileParser.getFilepath();
-    const cssImports = getCssFilesFromDependencyTree(this.dependencyTree);
+    const styleImports = getStyleFilesFromDependencyTree(this.dependencyTree);
```

Comment:
can we update tests to check that we get SCSS files too now?

Reply:
added tests!